src/site/stages/build/process-cms-exports/schemas/raw/node-landing_page.js

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable camelcase */
+
+module.exports = {
+  type: 'object',
+  properties: {
+    title: { $ref: 'GenericNestedString' },
+    changed: {
+      type: 'array',
+      maxItems: 1,
+      items: {
+        type: 'object',
+        properties: {
+          value: { type: 'string' },
+          format: { type: 'string' },
+        },
+        required: ['value', 'format'],
+      },
+    },
+    path: { $ref: 'RawPath' },
+    field_administration: {
+      type: 'array',
+      maxItems: 1,
+      items: { $ref: 'EntityReference' },
+    },
+    field_alert: { type: 'array' },
+    field_intro_text: { $ref: 'GenericNestedString' },
+    // This is essentially a list of paragraph-link_teaser,
+    // but only containing the data in field_link
+    field_links: {
+      type: 'array',
+      // I don't know that this array will *always* have only one item,
+      // but this schema passed the content build so it seemed
+      // like a decent idea to call it out as a potential change
+      // for the templates or CMS export
+      maxItems: 1,
```

Comment:
:+1: For what it's worth, other entities have had multiple `field_links` items. :man_shrugging: 

It's easy to change later. That's the beauty of these transformers. :smile: